### ISSUE TITLE
What is expected 'release' date for vefa-validator

### ISSUE THREAD

**Issue by aaron-kumar** [difi/vefa-validator#4]
What is expected 'release' date for vefa-validator. I am not asking about RC or Pre-release candidates

**Comment by m4caron** [comment]
To not open another issue, i will add another question that interests me : 
Is there a method currently in the new validator to retrieve the actual warning/error messages? 
I would like to fetch some more info than status flag : OK/WARN/ERROR.

Thank you.

**Comment by knyyy** [comment]
@m4caron 
I'm not sure if it exists methods to fetch error/warn messages directly from Report class, but I have done it this way:
```java
Validation validation = validator.validate(is);
Report report = validation.getReport(); 
if(report.getFlag() != FlagType.OK){ 
  for (SectionType sectionType : report.getSection()) {
    if(sectionType.getFlag() != Severity.OK) {
      for(AssertionType at : sectionType.getAssertion()){
        if(at.getFlag() != Severity.OK) {
          //You have more e.g. getTest, getFlag, getLocation.
          log.debug("ID: " + at.getIdentifier());
          log.debug("Description : " + at.getText());
       }
     }
   }
 }
```

I'm not sure if this approach is the best or correct , but it works. 
Hope this helps and I am open for comments/improvements.

**Comment by m4caron** [comment]
Thanks a lot Kennet for a great suggestion.

**Comment by klakegg** [comment]
The validator will be released in 2.0.0 when ready.

@m4caron This is an issue tracker were we keep to topic.
